docs/UserGuide.md

```diff
@@ -50,16 +50,18 @@ Avengers Assemble (AA) is a **desktop app for managing contacts, optimized for u
+| p/      | Phone number of person     | Should only contain numbers, and it should be at least 3 digits long.                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| e/      | Email of person            | Should be of the format local-part@domain and adhere to the following constraints: The local part should only contain alphanumeric characters and these characters excluding the parentheses: (+ _ . -) It may not start or end with the special characters. The domain name is made up of domain labels followed by periods. The domain name must end with a domain label at least 2 characters long, have each domain label start and end with alphanumeric characters, and consist of alphanumeric characters separated only by hyphens, if any. |         
+| a/      | Address of person          | Can take any values, but should not be blank.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| [t/]    | Tags of person             | Should be alphanumeric.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| [m/]... | Matriculation ID of person | The first letter must be an uppercase 'A', followed by 7 numbers, and ending with an uppercase letter.                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
are the specifications for tags and matric flipped? Each person should only have one matric ID while it can have multiple tags.

Reply:
Good catch pointing this potential issue out